src/test/java/core/basesyntax/service/FileReaderTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderTest {
+    private static final String CORRECT_PATH = "src/test/resources/testInput.csv";
+    private static final String EMPTY_PATH = "src/test/resources/emptyInput.csv";
```

Comment:
```suggestion
    private static final String EMPTY_FILE = "src/test/resources/emptyInput.csv";
```

Reply:
Don't think this is necessary, but fixed